app-games/src/main/java/com/aptoide/android/aptoidegames/feature_apps/presentation/BonusSectionView.kt

```diff
@@ -84,64 +86,103 @@ fun BonusSectionView(
         }
       )
     )
-    Image(
-      imageVector = getPromoSection(
-        iconColor = Palette.Primary,
-        outlineColor = Palette.Black,
-        backgroundColor = Palette.Secondary,
-        themeColor = Palette.Black
-      ),
-      contentDescription = "Bonus Section",
-      modifier = Modifier.fillMaxWidth(),
-      contentScale = ContentScale.FillWidth,
+    Text(
+      modifier = Modifier.padding(start = 16.dp, end = 40.dp, top = 8.dp, bottom = 8.dp),
+      text = annotatedString,
+      inlineContent = inlineContent,
+      style = AGTypography.InputsM,
+      color = Palette.White,
+      maxLines = 2,
+    )
+    BonusBundleView(
+      bundle = bundle,
+      navigate = navigate
     )
-    Row(
-      modifier = Modifier.fillMaxWidth(),
-      horizontalArrangement = Arrangement.SpaceBetween
-    ) {
-      BonusBannerText(
-        annotatedString = annotatedString,
-        inlineContent = inlineContent
+  }
+}
+
+@Composable
+fun BonusSectionHeader(
+  onClick: () -> Unit
+) {
+  Row(
+    modifier = Modifier.height(40.dp),
+    horizontalArrangement = Arrangement.SpaceBetween,
+  ) {
+    Box(modifier = Modifier.weight(1f, fill = false)) {
+      Image(
+        imageVector = getPromoSection(
+          backgroundColor = Palette.Secondary,
+          themeColor = Palette.Black
+        ),
+        contentDescription = "Bonus Section",
+        modifier = Modifier.fillMaxWidth(),
+        contentScale = ContentScale.FillWidth
       )
+      Row(
+        verticalAlignment = Alignment.CenterVertically,
+        horizontalArrangement = Arrangement.Start
+      ) {
+        Image(
+          modifier = Modifier
+            .padding(horizontal = 8.dp)
+            .requiredSize(40.dp)
+            .padding(all = 6.dp),
```

Comment:
It this padding required?

Reply:
I think so, since it. needs to be 40x40 but the present icon itself needs to have that smaller size (in figma it's 26.77 but I put it 28x28 this way)